src/sonic-config-engine/minigraph.py

```diff
@@ -211,7 +211,8 @@ def parse_dpg(dpg, hname):
 
             # If this VLAN requires a DHCP relay agent, it will contain a <DhcpRelays> element
             # containing a list of DHCP server IPs
-            if vintf.find(str(QName(ns, "DhcpRelays"))) is not None:
+            if vintf.find(str(QName(ns, "DhcpRelays"))) is not None and
```

Comment:
Line break in the middle of `if` statement requires parentheses around entire statement or backslash to escape newline. This is the cause of the check build failures.

Reply:
you're right. Refactored